frontend/src/metabase/dashboard/reducers.js

```diff
@@ -254,7 +266,7 @@ const dashcardData = handleActions(
       next: (state, { payload: { cardId, dashcardId } }) =>
         assocIn(state, [dashcardId, cardId]),
     },
-    [RESET]: { next: state => ({}) },
```

Comment:
From a quick search, I couldn't find that this action is being dispatched anywhere
https://github.com/metabase/metabase/blob/5ce3ba19b1ec3de4eff8ba58eaac1efd0d9c3a05/frontend/src/metabase/dashboard/actions/core.js#L7

Is it safe to say we could remove code that handles `RESET` action from all reducers now (definitely not in this PR, but to remove the redundancy)?

Reply:
Oh well... this is a huge problem of implicitly importing all named exports and using them with destructuring like this.
https://github.com/metabase/metabase/blob/a91d1e37d1063b260cb0489903f2b13eff033c0a/frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.jsx#L112

I think I've hit this same situation a few times now where functions are exported in a file but it's really hard to determine if it's being used. I think it might be better to try to be more verbose here (`...dashboardActions`) and list out all actions we want to map to a component. What do you think? This could be a future work of course.